### DISCUSSION TITLE
When comes the new Release 2.3.8

### DISCUSSION THREAD

**Discussion by Swerner1975** [mebjas/html5-qrcode#737] [General]
Hello,

we need the Issue Fix "Fix failed to parse sourcemap (issue #396)" very soon. 
Is it already foreseeable when the new release 2.3.8 will be available to retrieve via npm?

Best regards!

**Comment by mebjas** [comment]
Will push one this weekend!

**Comment by Swerner1975** [comment]
Any Update on releasing the new Release? 
Weekend and Week is over ... ;)

**Comment by mebjas** [comment]
Thanks for the patience, `v2.3.8` is published.

 - https://github.com/mebjas/html5-qrcode/releases/tag/v2.3.8
 - https://www.npmjs.com/package/html5-qrcode/v/2.3.8

**Summary**

### Version 2.3.8
 -   `Scanner Paused` color tone changes.
 -   Misc code refactor.
 -   Backwards compatibility improvement by [bilal-elchami@](https://github.com/bilal-elchami) in [PR#700](https://github.com/mebjas/html5-qrcode/pull/700).
 -   Documentation generated and moved to [scanapp.org/html5-qrcode-docs](https://scanapp.org/html5-qrcode-docs/)
 -   `Html5QrcodeScannerConfig` exported, thanks to [PR#728](https://github.com/mebjas/html5-qrcode/pull/728) by [seyfer@](https://github.com/seyfer).
 -   `LitExample` added, thanks to [PR#704](https://github.com/mebjas/html5-qrcode/pull/704) by [kusigit@](https://github.com/kusigit).

**Sponsors**
Huge thanks to supporters of the mission who sponsored the project (See [Leaderboard](https://ko-fi.com/minhazav/leaderboard)).

-   Jari
-   Dave from Gravity Wiz

See the [sponsorship page](https://ko-fi.com/minhazav) at ko-fi.

**Reply by mebjas** [reply]
Some relevant information for future releases: https://github.com/mebjas/html5-qrcode/discussions/747

**Comment by lakshay2711** [comment]
I am facing similar issue, can anyone pls let me know how to fix it ?
